Network/sent-received.1s.sh

```diff
@@ -15,7 +15,7 @@ TMPFILE="${TMPDIR}/bitbar-sent-received"
 
 if [[ -r "${TMPFILE}" ]]
 then
-    read -ra offsets < "${TMPFILE}"
+    read offsets < "${TMPFILE}"
```

Comment:
The `-r` here are not the same as the `-r` in the `if` statement.  In this case it does 'raw' handling of input vs checking if the file is readable

Reply:
https://travis-ci.org/matryer/bitbar-plugins/builds/640097794#L1011